src/Symfony/Bridge/Twig/AppVariable.php

```diff
@@ -89,6 +91,8 @@ public function getUser()
         if (null === $this->tokenStorage) {
             if (null === $this->container) {
                 throw new \RuntimeException('The "app.user" variable is not available.');
+            } elseif (!$this->container->has('security.context')) {
+                return;
```

Comment:
I don't know. It was an easy way to enable or hide sections in templates, depending on the application configuration without much effort. Apps will require to check themselves if security is configured and inject the result in every template where they need it, or create their own global.
When does returning `null` in such case became wrong ? I can't find anything mentioning the reasons in https://github.com/symfony/symfony/pull/13354